app/assets/stylesheets/report.scss

```
@@ -1,5 +1,13 @@
 .section.report-form{
   text-align: left;
+  padding-top: 10%;
+  min-height: 510px;
```

```suggestion
  padding-top: 16px;
```
Setting these makes the page look rather odd, let's have a smaller top padding and then use the body min height to make the page full size :)